ui/src/app/components/runtime-configuration-panel.tsx

```diff
@@ -516,6 +517,11 @@ const PanelMain = fp.flow(
       return warningDivs;
     };
 
+    // https://precisionmedicineinitiative.atlassian.net/browse/RW-9167
+    // We are disabling disabling standard disk option to encourage all users to sart using persistant disk
+    const disableCreate =
+      !disableDetachableReason &&
+      analysisConfig.diskConfig.detachableType !== DiskType.Standard;
```

Comment:
unfortunately this is not the right kind of "standard disk" - this is actually a standard PD.